src/styles/blocks/hand.scss

```diff
@@ -0,0 +1,55 @@
+.hand {
+  box-sizing: border-box;
+  transform-origin: bottom;
+  position: absolute;
+  left: 50%;
+  transform: translateX(-50%);
+
+  &__minute {
+    width: $minuteWidth;
+    height: $minuteHeight;
+
+    border: 3px solid #89CFEF;
+
+    top: 50% - $minuteHeight;
+    left: (100% - $minuteWidth) /2;
+
+    animation: rotate 60s linear infinite;
+  }
+
+  &__second {
+    width: $secondWidth;
+    height: $secondHeight;
+
+    border: 3px solid #98fb98;
+
+    top: 50% - $secondHeight;
+    left: (100% - $secondWidth) /2;
+
+    animation: rotate 5s linear infinite;
+  }
+
+  &__point {
+    background: #e6e600;
+    border-radius: 50%;
+
+    width: $pointWidth;
+    height: $pointWidth;
+
+    position: absolute;
+
+    top: 50%;
+    left: 50%;
+
+    transform: translate(-50%, -50%);
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)